_posts/2019/2019-05-04-react-hooks-what-all-the-fuss-is-about.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: React Hooks - What all the fuss is about
+date: 2019-05-04
+header_image: public/react-hooks.jpg
```

Comment:
Is this image from a free platform, such as pixabay, pexels, or unsplash?

If yes, I've sent you the right format via Mattermost. The current one doesn't fit.

Reply:
Yes, it's from unsplash.

Thanks, I updated it.